config/software/datadog-agent.rb

```diff
@@ -10,6 +10,12 @@
   source path: ENV['LOCAL_AGENT_REPO']
 end
 
+if ohai['platform'] == "windows"
+  integrations_dir="../integrations-core"
```

Comment:
I can do whatever you want, and yea that's definitely something that's a good cosmetic issue